### ISSUE TITLE
Add support for links inside properties

### ISSUE THREAD

**Issue by artyom-smushkov** [amake/orgro#128]
Some emacs packages like org-node or org-super-links add backlinks to node property called `BACKLINKS`. It would be great if orgro could handle such links the same way it handles them in node text.

**Comment by amake** [comment]
That links aren't parsed in properties is an oversight.

Are they just normal links?

**Comment by amake** [comment]
Rich content in drawer properties will be available in v1.51.0. It might take a bit to become available due to the holidays:

- [iOS](https://testflight.apple.com/join/8YTxcuZC)
- [Android](https://play.google.com/store/apps/details?id=com.madlonkay.orgro)

**Comment by artyom-smushkov** [comment]
Yes, they are just normal org links, in case of org-node id links, with org-super-links I'm not sure.

**Comment by artyom-smushkov** [comment]
I just checked latest release, links in properties work great, thanks!

**Comment by amake** [comment]
This is now released on all platforms.
